### ISSUE TITLE
Feature request: Add advertising

### ISSUE THREAD

**Issue by m1ga** [whilu/LMBluetoothSdk#9]
Would it be possible to add gatt advertising to the library?
Eg. http://blog.csdn.net/u013606170/article/details/46038283
including custom characteristics.

**Comment by whilu** [comment]
Yes, but this usage scenario doesn't often appear, so this feature isn't in this library. Maybe I should add this feature into this library in feature. Thanks!
